blocks/app-for-u/app-for-u.css

```diff
@@ -0,0 +1,106 @@
+.app-for-u {
+    background-color: #683d83;
+    padding: 60px 20px;
+    color: #fff;
+    display: flex;
+    flex-wrap: wrap-reverse;
+    justify-content: center;
+}
+
+.app-for-u__text-container {
+    width: 100%;
+}
+
+.app-for-u__title {
+    margin: 0;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 50px;
+    line-height: 59px;
+    letter-spacing: -0.01em;
+    text-transform: uppercase;
+    margin-bottom: 30px;
+    max-width: 335px;
+}
+
+.app-for-u__paragraph {
+    margin: 0;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 28px;
+    margin-bottom: 30px;
+}
+
+.app-for-u__image {
+    width: 100%;
+    max-width: 242.57px;
+    margin-bottom: 60.72px;
+}
+
+.app-for-u__links-container {
+    display: flex;
+    align-items: center;
+    padding: 20.87px 0 0;
+}
+
+.app-for-u__links-container :not(:last-child) {
+    margin-right: 23.21px;
+}
+
+.app-for-u__link-ios {
+    max-width: 117.92px;
+}
+
+.app-for-u__link-android {
+    max-width: 165px;
+}
+
+@media (min-width: 480px) {
+}
+
+@media (min-width: 630px) {
+}
+
+@media (min-width: 768px) {
+    .app-for-u {
+        padding: 100px 43px;
+    }
+    .app-for-u__title {
+        margin-bottom: 32px;
+        max-width: 100%;
+    }
+    .app-for-u__paragraph:not(:last-child) {
+        margin-bottom: 32px;
+    }
+}
+
+@media (min-width: 1024px) {
+    .app-for-u {
+        display: grid;
+        grid-template-columns: 2fr 1fr;
+        padding: 80px 0 90.87px 42px;
```

Comment:
Some thoughts about this breakpoint: https://www.loom.com/share/afa1108c3bf74785a9b57031bab33782

This is all an opinion, it all fits the brief so if you wanna skip this just let me know, but it might look nice to have this block break horizontal at a smaller breakpoint? I show it in the video above. Essentiall it'd be reducing this breakpoint to around 768px (or even lower) and adjust the 'padding' and 'box-sizing' properties on the paragraph box. Again, just an opinion, it already looks fantastic :)

Reply:
Thank you. It's better to adjust on smaller breakpoint 768px to horizontal. But I'm not sure on breaking to horizonal on lower screen 580px, it seems inconsistent/ a bit crowded to me considering with other sections on the same page/breakpoint. I'll add picture in slack group for better explaination.